website/www/site/content/en/blog/beam-2.49.0.md

```diff
@@ -51,7 +51,7 @@ For more information on changes in 2.49.0, check out the [detailed release notes
 ### Known Issues
 
 * Long-running Python pipelines might experience a memory leak: [#28246](https://github.com/apache/beam/issues/28246).
-
+* Python SDK's cross-language Bigtable sink mishandles records that don't have explicit timestamps set: [#28632](https://github.com/apache/beam/issues/28632)
```

Comment:
How about this?

The Python SDK's cross-language Bigtable sink does not properly handle records that do not have an explicit timestamp set. To avoid this issue, set explicit timestamps for all records before writing them to Bigtable.

Reply:
Adjusted it, PTAL